code/modules/organs/organ_internal.dm

```diff
@@ -229,6 +229,8 @@
 	removed_type = /obj/item/organ/internal/appendix
 
 /datum/organ/internal/proc/remove(var/mob/user, var/quiet=0)
+
+
```

Comment:
Der. Absoluten. Fukken.

Reply:
I agree.  It is a shame that the original stomach code PR had these unatomic fixes in them.